docs/source/get-started/suggested-models.md

```diff
@@ -0,0 +1,173 @@
+# Suggested Models
+
+In theory, Lumigator supports any model that is uploaded to the [Hugging Face Hub](https://huggingface.co/models?pipeline_tag=summarization&sort=trending)
```

Comment:
```suggestion
Lumigator supports any model that is uploaded to the [Hugging Face Hub](https://huggingface.co/models?pipeline_tag=summarization&sort=trending)
```

nit: We won't only support summarization in the future also, maybe change if we'd like this to be more general?

Reply:
sounds good, but i wouldn't include "To ensure a seamless experience," because that's not true...yet 😆

Reply:
Is this better?

> Lumigator supports any model uploaded to the [Hugging Face Hub](https://huggingface.co/models?pipeline_tag=summarization&sort=trending) and trained for summarization, provided the model is compatible with the required library versions (e.g., Transformers, vLLM, etc.). Practical factors such as compute availability and system configurations may also impact the successful use of a model. To get started, we have extensively tested a few models and created an endpoint to easily retrieve them.

Reply:
👍